internal/services/iotcentral/iotcentral_application_resource.go

```diff
@@ -73,12 +72,11 @@ func resourceIotCentralApplication() *pluginsdk.Resource {
 				Type:     pluginsdk.TypeString,
 				Optional: true,
 				ValidateFunc: validation.StringInSlice([]string{
-					string(iotcentral.F1),
-					string(iotcentral.S1),
-					string(iotcentral.ST1),
-					string(iotcentral.ST2),
+					string(apps.AppSkuSTOne),
+					string(apps.AppSkuSTTwo),
+					string(apps.AppSkuSTZero),
 				}, false),
-				Default: iotcentral.ST1,
+				Default: apps.AppSkuSTOne,
```

Comment:
fwiw these should be:

```suggestion
				Default: string(apps.AppSkuSTOne),
```